_posts/2017-08-11-an-interface-between-the-natural-and-the-artificial.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+name: an-interface-between-the-natural-and-the-artificial
+title: An Interface Between The Natural And The Artificial
+date: 2017-08-11 08:00:00 +00:00
+author: Luciano Palma
+image:
+    src: /assets/img/custom/blog/melbourne-sundial.jpg
+    attribution:
+      text: (Resized) Picture by Jeepika (CC BY-SA 3.0)
+      href: https://en.wikipedia.org/wiki/File:Melbourne_sundial_at_Flagstaff_Gardens.JPG#/media/File:Melbourne_sundial_at_Flagstaff_Gardens.JPG
+tags:
+- refactoring
+- artificial-systems
+- interface
+---
+
+Everything built by humans is meant to work within a proper environment. Herbert A. Simon, in his book [The Sciences of the Artificial][4], defines things that are created or manipulated by humans as artificial systems. These systems are composed of an inner environment and they interact with the outer environment, that is, the external world, through an interface.
+
+Examples of artificial systems may include cars, a cotton farm and a computer program. These systems, even biological ones, exist only to serve humans a pre-defined purpose. Dogs, for instance, are the result of thousands of years of taming and selective breeding, primarily intended for hunting and to protect our farms, but now also used as family pets. They only exist because we decided to, also most of the current breeds would not survive in the natural environment, where natural selection and survival of the fittest are the main criterias.
+
+Cars were created primarily to be used by humans as a vehicle of transport on the ground. Thus, In order to fulfil its purpose, a car cannot be placed under water or suspended in the air. It was built with an interface which will only interact properly with an environment consisting of, for example, a paved road.
+
+
+##The artificial environment in which we live
```

Comment:
Some markdown processors require a space between the header mark and the text. `## The artificial ...`